Source/SA/RFmx WLAN/RFmxWLAN.cs

```diff
@@ -213,11 +214,11 @@ public struct TxPServoResults
             sessionHandle.SetLOSource("", commonConfig.LOSource);
             sessionHandle.SetDownconverterFrequencyOffset("", commonConfig.LOOffset);
             wlanSignal.SetSelectedPorts(selectorString, commonConfig.SelectedPorts);
-            wlanSignal.ConfigureDigitalEdgeTrigger(selectorString, commonConfig.DigitalEdgeSource, commonConfig.DigitalEdgeType, commonConfig.TriggerDelay_s, commonConfig.EnableTrigger);
+            wlanSignal.ConfigureDigitalEdgeTrigger(selectorString, commonConfig.DigitalEdgeSource, commonConfig.DigitalTriggerEdge, commonConfig.TriggerDelay_s, commonConfig.EnableTrigger);
             wlanSignal.ConfigureFrequency(selectorString, commonConfig.CenterFrequency_Hz);
             wlanSignal.ConfigureExternalAttenuation(selectorString, commonConfig.ExternalAttenuation_dB);
 
-            if (autoLevelConfig.AutoLevelReferenceLevel) wlanSignal.AutoLevel(selectorString, autoLevelConfig.AutoLevelMeasureTime_s);

```

Comment:
i don't see a removal of the auto level config struct in the diff. did you take it out?

Reply:
No, I left the struct itself in place because the Power Servo function uses it and I didn't want to waste time messing with that function since it will shortly be deprecated. I thought to remove it when that was pulled out.

Reply:
I see. Thoughts on adding an `[Obsolete]` attribute until it's removed? That would help remind us to take it out.

Reply:
Hmm. Well, it's not technically obsolete (yet) haha, since we don't have anything to replace it with. How about I make an issue to make sure that we remember?

Reply:
i can dig it